tools/sylph/tool_data_table_conf.xml.test

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<tables>
+    <!-- Locations of Sylph database in the required format -->
+    <table name="sylph_databases" comment_char="#">
+        <columns>value, name, path</columns>
```

Comment:
Can we add a 4-column here, maybe called "version" - just in case sylph changes the DB format at some point.